paasta_tools/setup_kubernetes_cr.py

```diff
@@ -162,12 +163,19 @@ def setup_all_custom_resources(
         kube_client.apiextensions,
         kube_client.apiextensions_v1_beta1,
     ]:
-        cluster_crds = {
-            crd.spec.names.kind
-            for crd in apiextension.list_custom_resource_definition(
+
+        try:
+            crds_list = apiextension.list_custom_resource_definition(
                 label_selector=paasta_prefixed("service")
             ).items
-        }
+        except ApiException:
+            log.debug(
+                "Listing CRDs with apiextensions/v1beta1 not supported on this cluster, falling back to v1"
```

Comment:
this is the other way around in this function, right? we try v1 first and then fall back to v1beta1?

Reply:
you're right, actually we'll try/catch both ubt I agree we fallback to v1beta1